examples/fft_vs_geom_movie.py

```diff
@@ -0,0 +1,348 @@
+# Copyright (c) 2012-2016 by the GalSim developers team on GitHub
+# https://github.com/GalSim-developers
+#
+# This file is part of GalSim: The modular galaxy image simulation toolkit.
+# https://github.com/GalSim-developers/GalSim
+#
+# GalSim is free software: redistribution and use in source and binary forms,
+# with or without modification, are permitted provided that the following
+# conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions, and the disclaimer given in the accompanying LICENSE
+#    file.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions, and the disclaimer given in the documentation
+#    and/or other materials provided with the distribution.
+#
+
+"""@file fft_vs_geom_movie.py
+Script to demonstrate the geometric approximation to Fourier optics.  Simulates both a randomly
+evolving set of Zernike aberrations and a frozen-flow Kolmogorov atmospheric phase screen.
+
+To visualize only the Zernike aberrations, add `--nlayers 0` as a command line argument.
+
+To visualize only atmospheric effects, add `--sigma 0` as a command line argument.
+
+Some suggested command lines, appropriate for LSST:
+
+Optics only simulation:
+  $ python fft_vs_geom_movie.py --diam 8.36 --obscuration 0.61 --sigma 0.05 --nlayers 0
```

Comment:
I didn't mean to foist my problem on you - I was mainly asking to establish whether it really is just my system-dependent issue (vs. misuse of script, or a generic python 2.7 issue).  Thanks for offering but I will look into it myself. :)